src/eckit/io/Buffer.h

```diff
@@ -40,6 +40,8 @@ class Buffer : private NonCopyable {
 
     ~Buffer();
 
+    std::string_view view() const noexcept { return std::string_view(buffer_, size_); }
+
```

Comment:
`Buffer` is a byte buffer and while it could be used as a c-string it think this should not be done. Especially. since the only usage I can see is in tests.